core/controllers/admin.py

```diff
@@ -245,6 +250,19 @@ def post(self):
                 result = {
                     'opportunities_count': opportunities_count
                 }
+            elif self.payload.get('action') == 'update_feature_flag_rules':
+                feature_name = self.payload.get('feature_name')
+                new_rule_dicts = self.payload.get('new_rules')
+                commit_message = self.payload.get('message')
```

Comment:
Prefer ``commit_message`` for the message payload key for consistency & to contextualize what 'message' corresponds to.